DIRECTORY.md

```diff
@@ -394,6 +394,7 @@
   * Tests
     * [Test Min Spanning Tree Kruskal](graphs/tests/test_min_spanning_tree_kruskal.py)
     * [Test Min Spanning Tree Prim](graphs/tests/test_min_spanning_tree_prim.py)
+  * [Topological Warshall](graphs/topological_warshall.py)
```

Comment:
You don't need to update the DIRECTORY.md file, it is auto generated periodically.